src/dto/connection.py

```diff
@@ -1,30 +1,47 @@
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
+from src.dto import connection_config
 
-DATABASE_URI = "localhost"
-PORT = ":3306"
-DATABASE_NAME = "test"
-USERNAME = "root"
-PASSWORD = "root"
-MYSQL_CONNECTION = "mysql://" + USERNAME + ":" + PASSWORD + "@" + DATABASE_URI + PORT + "/" + DATABASE_NAME
 
 app = Flask(__name__)

```

Comment:
Co tutaj robi linijka do odpalania flaska tak właściwie ?

Reply:
Tego nie wiem. Wydaje mi sie ze to cos w stylu default mialo byc ale bym to wyrzucil

Reply:
no to już @Yurati trzeba spytać, ale wydaje mi się, że to jest do testów, które nie wiem czy wogoóle bedziemy robić :D :D

Reply:
To było do testów tylko i to zostawiłem do table testowych, do wyrzucenia